backend/api/views.py

```
@@ -41,6 +41,9 @@ def addServer(request):
     body = json.loads(request.body)
     if 'serverId' not in body:
         return Response("The server id is required", status=400)
+    duplicate = Server.objects.filter(serverId=serverId).exists()
```

Ruff

`Server` may be undefined, or defined from star imports

(F405)